docs/Developers/Meta-review.rst

```diff
@@ -0,0 +1,140 @@
+Meta-Reviewing
+==============
+
+This document is a guide to coala's meta-review process.
+
+What is Meta-review?
+--------------------
+
+People respond to review comments on pull requests by giving feedback. Emoji
+make it much easier to give and receive feedback without a long comment thread.
+Those emoji are called
+`reactions <https://developer.github.com/v4/enum/reactioncontent/>`__ on GitHub
+or `award emoji <https://docs.gitlab.com/ee/user/award_emojis.html>`__ on GitLab.
+They are used by people to publicly express their feelings about review
+comments, which provides feedback to comment authors and all other members.
+
+There are 6 types of reactions on GitHub:
+
+============  =============  =======  =====
+ Name         Markdown       Unicode  Emoji
+============  =============  =======  =====
+ THUMBS_UP    ``:+1:``       U+1F44D  👍
+ THUMBS_DOWN  ``:-1:``       U+1F44E  👎
+ LAUGH        ``:smile:``    U+1F604  😄
+ HOORAY       ``:tada:``     U+1F389  🎉
+ CONFUSED     ``:confused``  U+1F615  😕
+ HEART        ``:heart:``    U+FE0F   ❤️
+============  =============  =======  =====
+
+These reactions are reviews to reviews, thus we call them meta-reviews. To
+encourage people to do meta-reviews, we build a meta-review system, which
+collects reactions, analyze them, and use that information to score people.
```

Comment:
"score people" is a bit off-putting, when simplified like that.

we use that information to  ..

- improve our process,
- improve the quality of reviews being done.

etc

We are giving a score to their reviews.